mimic/rest/maas_api.py

```diff
@@ -642,10 +642,21 @@ def view_agent_host_info(self, request, tenant_id):
                                'details': 'You must specify an agentId, entityId, or an entity URI.',
                                'mimicNotes': 'But mimic will only accept entityId right now',
                                'txnId': '.fake.mimic.transaction.id.c-1111111.ts-123444444.v-12344frf'})
+
         entity_id = request.args['entityId'][0].strip()
         for e in self._entity_cache_for_tenant(tenant_id).entities_list:
             if e['id'] == entity_id:
                 agent_id = e['agent_id']
+                break
+        else:
```

Comment:
the `if` block then the code outside the block and then the `else` is making this code not easy to read and comprehend.